index.bs

```diff
@@ -2020,9 +2020,16 @@ Consensus</h4>
 
 		<dt><dfn id="def-Dissent">Dissent</dfn>:
 		<dd>
-			At least one individual in the set registers a <a href="#FormalObjection">Formal Objection</a>.
+			At least one individual in the set sustains an objection.
```

Comment:
I think we're missing a class here. I agree with comments made elsewhere that there can be dissent by disagreement that is not escalated to Objection level. 

To make this more concrete, consider that a WG is faced with two competing proposals to resolve the same issue. There are no objections to either, but there is also a split of preferences amongst members. The Chair may reasonably assert a Decision that is the choice with the least negative preference, i.e. dissent.

Reply:
Hi. I think you may be missing that we have a difference of definition in the process between mere disagreement, and dissent (which is an FO or equivalent).  Maybe this suggests we should use "sustained dissent" instead of plain "dissent"?

Reply:
Bit puzzled about this @dwsinger. Disagreement isn't a defined term in the Process. The only relevant reference I can find is in §5. Decisions where individuals are encouraged (SHOULD) to register an FO if they disagree _strongly_ with a decision (my emphasis). 

This recommendation actually seems like an anti-pattern, since the goal is to resolve strong disagreements _before_ they need to be escalated to an FO. Maybe we need a more nuanced section about handling disagreements as well?

Reply:
disagreement doesn't need defining in the process, it has its usual meaning. Dissent is formally defined, however, to be an FO or its equivalent. So in the Process, not all disagreement is Dissent. As I say, we might make this clearer by using the phrase "sustained dissent".

Reply:
I don't see any problem using "sustained dissent", but I fear there's a logical problem with saying that disagreement has its usual meaning, and then also saying that disagreement _excludes_ (sustained) dissent. If it's important that the two categories are mutually exclusive then we would have to define "disagreement" more explicitly.

Reply:
I'd rather not rename the concept of “dissent” in the Process. This pull request is about making its definition (and consequently the definition of “consensus”) more broadly applicable than just to REC-track documents. If we want to rename concepts, that should be a separate issue...